<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StructureCompare</class>
 <widget class="QWidget" name="StructureCompare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0,0">
   <item row="1" column="0" colspan="4">
    <widget class="QTableView" name="structuresTable"/>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="structureColoringCombo"/>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="selectStructuresButton">
     <property name="text">
      <string>Select Structures</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Structure Coloring</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font-size: 24pt;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Structure Compare</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
